src/Homework4/task5.py

```diff
@@ -0,0 +1,40 @@
+""" Первая строка входных данных содержит количество школьников N.
+Далее идет N чисел Mi, после каждого из чисел идет Mi строк,
+содержащих названия языков, которые знает i-й школьник.
+
+Пример входных данных:
+
+3          # N количество школьников
+2          # M1 количество языков первого школьника
+Russian    # языки первого школьника
+English
+3          # M2 количество языков второго школьника
+Russian
+Belarusian
+English
+3
+Russian
+Italian
+French
+Выходные данные
+В первой строке выведите количество языков, которые знают все школьники.
+Начиная со второй строки - список таких языков. Затем - количество языков,
+ которые знает хотя бы один школьник, на следующих строках - список
+таких языков.
+"""
+
+
+Nbr_1 = 2       # введите количество языков которые знает первый школьник
+language_1 = ['Russian', 'English']
+Nbr_2 = 3       # введите количество языков которые знает второй школьник
```

Comment:
redundant comments. you variable name should tell what would be kept here, not comments